test/factories/projects_factory.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :project do
+    unit
+    campus
```

Comment:
Also need the student in here... check any other relation as well.

Reply:
Student user